docs/collaboration.md

```diff
@@ -0,0 +1,105 @@
+# Collaborative Publishing with Posit Publisher
+
+[Posit Publisher](https://github.com/posit-dev/publisher)—the extension to
+publish content to Posit Connect using
+[Visual Studio Code](https://code.visualstudio.com/) or
+[Positron](https://positron.posit.co/)—enables seemless collaboration. This
+workflow guide explains how to collaborate with others on a deployed piece of
+content.
+
+## Deployment and Configuration files
+
+Posit Publisher creates and utilizes two files for deploying content. Both are
+inside of the `.posit/publish` directory in the same directory as
+your entrypoint file.
+
+#### The Configuration file
+
+The Configuration file describes how the content will be deployed to Posit
+Connect.
+
+An example of a Configuration file is `/.posit/publish/content-title-061E.toml`.
+
+For more details on the Configuration file see the [Configuration file reference](./configuration.md).
+
+#### The Deployment file
+
+The Deployment file contains details on where the content was deployed and its
+state. Deployment files are automatically generated by Posit Publisher,
+updated during deploys, and not meant to be edited.
+
+An example of a Deployment file is
+`/.posit/publish/deployments/deployment-58A3.toml`.
+
+Utilizing both of these files, alongside a credential and collaborator access
+on Posit Connect, you and your collaborators can update and configure a single
+piece of content.
```

Comment:
Would it make sense to be even more explicit here and say: you should generally not need to open or think about this file, though you *should* track it and make updates to it when git (or whichever RCS you're using) tells you do?

Reply:
I re-wrote the intro for the Deployment file section a bit to be more explicit about how to treat the file. I didn't outline that it should be tracked since the details about source control or utilizing the content bundle are  detailed below.

> The Deployment file details where the content was deployed and its state.
Deployment files are automatically generated by Posit Publisher and updated
during deploys. They are not meant to be edited. Posit Publisher will take care
of this file for you so you do not need to think about it aside from the
collaboration tips below.